server/config/detailsValidation.test.js

```diff
@@ -0,0 +1,338 @@
+const config = require('./incident.js')
+const formProcessing = require('../services/formProcessing')
+
+const validatorChecker = formConfig => input => {
+  const { payloadFields: formResponse, errors } = formProcessing.processInput(formConfig, input)
+  return { formResponse, errors }
+}
+
+let validInput = {}
+const check = validatorChecker(config.details)
+beforeEach(() => {
+  validInput = {
+    positiveCommunication: 'true',
+    personalProtectionTechniques: 'true',
+    batonDrawn: 'true',
+    batonUsed: 'true',
+    pavaDrawn: 'true',
+    pavaUsed: 'true',
+    guidingHold: 'true',
+    guidingHoldOfficersInvolved: 2,
```

Comment:
this should be a string: `'2'`, as it will be sanitised to an integer

Reply:
done